ci-operator/config/openstack-k8s-operators/dataplane-operator/openstack-k8s-operators-dataplane-operator-main.yaml

```diff
@@ -0,0 +1,69 @@
+base_images:
+  golangci-lint:
+    name: golangci-lint
+    namespace: ci
+    tag: v1.46.0
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-golang-1.18-openshift-4.12
+images:
+- dockerfile_literal: |
```

Comment:
Hello Fabricio,

Recently we have created a pre-commit image maintained on openstack-k8s-opertors-ci repo[1] to avoid build pre-commit image on every PR and duplicating this dockerfile_literal in every repo.

openstack-operator already uses this prec-commit as base image, and I have just proposed patch for nova and placement operator as well. Could you please check [2] and [3] as an example and consume the already create pre-commit image as base container to run pre-commit job


[1] https://github.com/openstack-k8s-operators/openstack-k8s-operators-ci/blob/8991de5dc145964c1c8fb8c1e6fa7a31cee41e6b/images/pre-commit/Dockerfile
[2] https://github.com/openshift/release/pull/35761
[3] https://github.com/openshift/release/pull/35765

Reply:
yes, I'll follow it